docs/installing.rst

```diff
@@ -0,0 +1,41 @@
+=================================================
+Installing
```

Comment:
Perhaps Getting Started? pi-topOS users won't need to do this, so we can say "use pi-topOS" before recommending manual install

Reply:
Feel like we should add this "getting started" section to the index page; lets discuss tomorrow

Reply:
I agree - fewer sections, and it's expected

Reply:
Actually, I'm not sure - we would lose it from the side bar, which would be a shame. It's not obvious that you need to click the home button in order to find that again